npm/angular/cypress.config.ts

```diff
@@ -11,13 +12,9 @@ export default defineConfig({
       return require('./cypress/plugins')(on, config)
     },
     devServer (cypressDevServerConfig) {
-      const { startDevServer } = require('@cypress/webpack-dev-server')
       const webpackConfig = require('./cypress/plugins/webpack.config')
 
-      return startDevServer({
-        options: cypressDevServerConfig,
-        webpackConfig,
-      })
+      return devServer(cypressDevServerConfig, { webpackConfig })
```

Comment:
Is this intentional? Since it could be done more idiomatically like so (with the `require` at the top):

```js
component: {
  devServer,
  devServerConfig: { webpackConfig }
}
```